metrics_ci_integration/lib/jenkins/model/jenkins_build_artifact.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:ci_integration/jenkins/model/jenkins_build.dart';
+import 'package:equatable/equatable.dart';
+
+/// A class representing a single artifact of a [JenkinsBuild].
+class JenkinsBuildArtifact extends Equatable {
+  /// A name of this artifact file.
+  final String fileName;
+
+  /// A path to this artifact.
+  final String relativePath;
```

Comment:
This is a property name from Jenkins API. Since it has no documentation I cannot answer this question. 

I suppose that it stands for the relative path in artifacts tree for the build. From the root to the artifact file.